dashboards/mongodb/metadata.yaml

```diff
@@ -1,9 +1,11 @@
 sample_dashboards:
   -
     category: MongoDB
-    id: mongodb-gce-overview
-    display_name: MongoDB GCE Overview
-    description: "This dashboard has several charts for the related [Google Cloud Ops Agent Integration for MongoDB](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/mongodb), including metrics like Active Connections, Collection Size, Server Memory Usage, Index Count, Index Size, and Operations."
+    id: overview
+    display_name: MongoDB Overview
+    description: |-
+      This dashboard is based on [Google's Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/mongodb#monitored-metrics) and has charts displaying Number of Connections, Number of Sessions, Number of Databases, Data Size, Storage Size, Number of Objects, Number of Collections, Number of Document Operations, Number of Indexes, Number of Indexes accessed, Index Size, Number of requests, Received Bytes, Transmitted Bytes, Operations Rate Avg., Operations Duration, Memory Usage, Cache Hits / Misses, Global Lock Hold Time, Number of Cursors, and Cursor Duration from MongoDB as well as charts of infrastructure related metrics for the running MongoDB VMs: CPU % Top 5 VMs, Memory % Top 5 VMs, and Hosts by Region.
```

Comment:
I think `Number of x` can be replaced with `x Count` here, or `Number of` could just be dropped

Reply:
updated!